satellizer.js

```diff
@@ -852,8 +819,11 @@
           }
         };
       }])
-    .config(['$httpProvider', function($httpProvider) {
-      $httpProvider.interceptors.push('satellizer.interceptor');
+      .config(['$httpProvider', 'satellizer.config', 'localStorageServiceProvider', function($httpProvider, config, localStorageServiceProvider) {
+        $httpProvider.interceptors.push('satellizer.interceptor');
+        localStorageServiceProvider.prefix = config.tokenPrefix;
```

Comment:
use setters as defined in the official documentation https://github.com/grevory/angular-local-storage#setprefix

```
.config(function (localStorageServiceProvider) {
  localStorageServiceProvider.setPrefix(config.tokenPrefix);
  localStorageServiceProvider.setStorageType(config.storage);
});
```